proposals/infra/INF-0004-validator-hashing.md

```diff
@@ -210,21 +196,22 @@ runtime.
 > these legacy zero'd has shaders will not run have the validator invoked on them
 > by the runtime/debug layer since that wasn't expected of them.
 
-#### D3D Runtime Validation Control
+#### D3D Debug Layer Validation Control
 
-Apps can configure how the runtime validates bytecode passed to it. 
+Apps can configure how the debug layer runtime validates bytecode passed to it. 
 
 The validation referred to here is implementation exposed by `dxil.dll` 
-or `dxcompiler.dll`, and also embedded in `d3d12sdklayers.dll`.  
+or `dxcompiler.dll`, and also embedded in the debug layer (`d3d12sdklayers.dll`).
+The implementation in the debug layer is used by default, but the external ones 
+can be requested.
+
 This is the same validation used when the compiler endorses bytecode at 
 compile time by applying a hash.  So it would typically be 
 redundant to validate again, but it can be useful to validate shaders that have 
 the `BYPASS` or `PREVIEW_BYPASS` hash, or have the option to force validation. 
 
-If there is bytecode to validate, the runtime and/or debug layer attempt to 
-use a validator implementation from one of these dlls. Configuration flags 
-determine the order of preference for implementation and whether it is ok to 
-skip validation if an implementation is not found.
+If the debug layer finds errors when running the bytecode validator it only 
+reports messages, but doesn't fail shader creation.
 
 The following flags in `d3d12.h` show the validation control options.  
 `ID3D12Device` exposes `ID3D12BytecodeOptions` to set the flags, shown 
```

Comment:
Should this configuration be set off the debug interface now, rather than the device?